<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequeteEleve</class>
 <widget class="QWidget" name="RequeteEleve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Requêtes sur la table Eleve</string>
  </property>
  <widget class="QComboBox" name="anneeComboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Année d'entrée</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2009</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2010</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2011</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="departementCheckBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Département</string>
   </property>
  </widget>
  <widget class="QPushButton" name="validerButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QComboBox" name="promoComboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Promotion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EI1PIP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EI3AMG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EI3ELE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EI3INF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EI3INI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EI3MEC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SL1MAI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SL1SCM</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="departementSpinBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>55</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>96</number>
   </property>
  </widget>
  <widget class="QLabel" name="resultatLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="annulerButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
